### PR TITLE
Reacting to KoreBuild xunit test changes

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json [hunk 1]
```diff
@@ -26,10 +26,12 @@
         "dotnet-test-xunit": "1.0.0-dev-*"
       }
     },
-    "dnx451": {
+    "net451": {
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json]: WebApiCompatShim gets low traffic for code changes. Is it ok to remove VS + DNX test support for this?

Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json]: cc @Eilon

Comment (Eilon) [inline test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json]: Fine with me. We can always add it back later.

File: test/Microsoft.AspNetCore.Mvc.FunctionalTests/MvcSampleFixture.cs [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
 {
     public class MvcSampleFixture<TStartup> : MvcTestFixture<TStartup>
     {
         public MvcSampleFixture()
-#if NETSTANDARDAPP1_5
-            :base("../../../../../../samples/")
-#else
-            : base("../../../../samples/")
-#endif
+            :base(Path.Combine("..", "..", "..", "..", "..", "..", "samples"))
```
Comment (Eilon) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/MvcSampleFixture.cs]: Formatting (space after `:`)

File: test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json [hunk 3]
```diff
@@ -26,10 +26,12 @@
...
+      "frameworkAssemblies": {
+        "System.Threading.Tasks": ""
```
Comment (Eilon) [inline test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json]: Does this need `{ "type": "build" }` like in https://github.com/aspnet/HttpAbstractions/pull/579?

Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json]: It's a test project, so it shouldn't matter. Plus I think `System.Threading.Tasks` is a real fx assembly so it's fine to list it unlike `System.Runtime`.

File: test/Microsoft.AspNetCore.Mvc.FunctionalTests/MvcSampleFixture.cs (hunk 2 above)
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/MvcSampleFixture.cs]: Yup. Fixed in actual commit.

File: test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json (hunk 3 above)
Comment (Eilon) [inline test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json]: Ok.

**Pull request closed without merging**